apiary.apib

```diff
@@ -954,11 +954,570 @@ Outputs the list of peers that were removed.
 
 # Group config
 
-## edit
+## config [POST /config{?arg1,arg2}{&bool,json}]
 
-## replace
+'ipfs config' controls configuration variables. It works
+much like 'git config'. The configuration values are stored in a config
+file inside your IPFS repository.
 
-## show
++ Parameters
+    + arg1: "Datastore.Path" (string, required) - The key of the config entry.
+    + arg2: "~/.ipfs/datastore" (string, required) - The value to set the config entry to.
+    + bool (boolean, optional) - Set a boolean.
+    + json (boolean, optional) - Parse stringified JSON.
+
++ Request Without Arguments
+
+    #### curl
+
+        curl -i -X POST "http://localhost:5001/api/v0/config"
+
+    + Body
+
+        ```
+        curl -i -X POST "http://localhost:5001/api/v0/config"
+        ```
+
++ Response 400
+
+    + Headers
+
+        ```
+        Date: Tue, 02 Feb 2016 16:37:00 GMT
+        Content-Length: 26
+        Content-Type: text/plain; charset=utf-8
+        ```
+
+    + Attributes (string)
+
+    + Body
+
+        ```
+        Argument 'key' is required
```

Comment:
Any reason for this response being a `string` instead of a json object like the other [error responses](https://github.com/ipfs/http-api-spec/blob/feature/config/apiary.apib#L1032-L1035)?
Can we make that more uniform?